content/docs/gke/deploy/deploy-ui.md

```diff
@@ -6,35 +6,37 @@ weight = 3
 
 This page provides instructions for using the Kubeflow deployment web app to
 deploy Kubeflow on GCP. If you prefer to use the
-command line, see the guide to 
-[deployment using the CLI](/docs/gke/deploy/deploy-cli).
+command line, see the guide to [deployment using the CLI](/docs/gke/deploy/deploy-cli).
 
-1. Open [https://deploy.kubeflow.cloud/](https://deploy.kubeflow.cloud/#/deploy)
+<ol>
```

Comment:
@jay-saldanha Thanks for these updates! Here's a summary of the work that still needs doing to finish off this PR:

* As it currently stands, this PR changes most of the page source from Markdown to HTML. We prefer Markdown for readability and easier maintenance, though we do sometimes use HTML within a Markdown file for complex tables, adding anchor tags, or other formatting that is difficult in Markdown. TODO: Revert the changes where HTML formatting is not required.

* There are also some formatting/syntax issues on the page - couple of review comments from the related PR https://github.com/kubeflow/website/pull/685 (now closed) haven't made it into this PR.

* As you mentioned, there are still some comments in the HTML that need resolving.